datalad/interface/tests/test_add_archive_content.py

```diff
@@ -215,48 +243,67 @@ def d1_basic_checks():
             ok_archives_caches(repo_path, 0)
 
         # and by default it just does it, everything goes to annex
-        repo_ = add_archive_content('1.tar.gz')
-        eq_(repo.path, repo_.path)
+        res = add_archive_content('1.tar.gz')
+        assert_in_results(
+            res,
+            action='add-archive-content',
+            status='ok'
+        )
         d1_basic_checks()
 
-        # If ran again, should proceed just fine since the content is the same so no changes would be made really
-        add_archive_content('1.tar.gz')
+        # If ran again, should proceed just fine since the content is the same
+        # so no changes would be made really
+        res = add_archive_content('1.tar.gz')
+        assert_in_results(
+            res,
+            action='add-archive-content',
+            status='ok'
+        )
 
-        # But that other one carries updated file, so should fail due to overwrite
+        # But that other one carries updated file, so should fail due to
+        # overwrite
         with assert_raises(RuntimeError) as cme:
-            add_archive_content(opj('1u', '1.tar.gz'), use_current_dir=True)
+            add_archive_content(Path('1u') / Path('1.tar.gz'),
+                                use_current_dir=True)
 
         # TODO: somewhat not precise since we have two possible "already exists"
         # -- in caching and overwrite check
         assert_in("already exists", str(cme.exception))
         # but should do fine if overrides are allowed
-        add_archive_content(opj('1u', '1.tar.gz'), existing='overwrite', use_current_dir=True)
-        add_archive_content(opj('2u', '1.tar.gz'), existing='archive-suffix', use_current_dir=True)
-        add_archive_content(opj('3u', '1.tar.gz'), existing='archive-suffix', use_current_dir=True)
-        add_archive_content(opj('4u', '1.tar.gz'), existing='archive-suffix', use_current_dir=True)
+        add_archive_content(Path('1u') / Path('1.tar.gz'), existing='overwrite',
```

Comment:
Here and elsewhere it would be sufficient to do this:

```suggestion
        add_archive_content(Path('1u') / '1.tar.gz', existing='overwrite',
```

Reply:
done, thx!